lightningd/json.c

```diff
@@ -129,6 +130,16 @@ void json_add_address(struct json_result *response, const char *fieldname,
 		json_add_string(response, "address", addrstr);
 		json_add_num(response, "port", addr->port);
 	}
+	else if (addr->type == ADDR_TYPE_TOR_V2) {
```

Comment:
Should put closing `}` in same line as `else` and `if`:

     } else if (addr->type == ADDR_TYPE_TOR_V2) {